contracts/tests/MaliciousSecurityAdapter.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// DPMAdapter.sol
+
+// Copyright (C) 2023 Oazo Apps Limited
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.0;
+
+import "../interfaces/IAdapter.sol";
+import "../McdView.sol";
+
+contract MaliciousSecurityAdapter is ISecurityAdapter {
```

Comment:
Is this used anywhere?

Reply:
yes

[getContractFactory](https://github.com/OasisDEX/automation-smartcontracts/blob/2f4dfa3fbf8580229f40c20a4a13c006c689e403/test/automation-bot.ts#L1010)